src/App.tsx

```diff
@@ -1,7 +1,28 @@
 import React from "react";
+import { BrowserRouter, Route, Routes, Navigate } from "react-router-dom";
+import NavBar from "./components/Navbar";
+import RegistrationPage from "./pages/RegistrationPage";
+import MainPage from "./pages/MainPage";
+import LoginPage from "./pages/LoginPage";
+import MyPage from "./pages/MyPage";
+import { useTypedSelector } from "./hooks/useTypedSelector";
 
 const App = () => {
-  return <div>Hello, world!</div>;
+  const { isLogin } = useTypedSelector((state) => state);
```

Comment:
This stuff definitely hasn't sense
```suggestion
  const isLogin = useIsLoginSelector((state) => state.isLogin);
```